src/components/FiltersList/index.tsx

```diff
@@ -10,18 +10,22 @@ export interface Filter {
   }[];
 }
 
-export interface FacetUpdatePayload {
+export interface AppliedFilters {
+  [filterName: string]: string[];
+}
+
+export interface FilterUpdatePayload {
   facetName: string;
```

Comment:
maybe this should be
```suggestion
  filterName: string;
```